py/selenium/webdriver/common/__init__.py

```diff
@@ -14,3 +14,20 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+from .keys import Keys
```

Comment:
i don't believe it's good form to put functionality in the init file. It seems more appropriate to put it in common/keys.py